src/UI/Internal/Utils/Element.elm

```diff
@@ -0,0 +1,14 @@
+module UI.Internal.Utils.Element exposing (style, tuplesToStyles)
+
+import Element exposing (Attribute)
+import Html.Attributes as HtmlAttrs
+
+
+style : String -> String -> Attribute msg
+style k v =
+    Element.htmlAttribute <| HtmlAttrs.style k v
+
+
+tuplesToStyles : ( String, String ) -> Attribute msg
```

Comment:
I renamed this function to be more explicit about what it does. WDYT?

Reply:
That's a pair, we may end up having a triple later.